docusaurus.config.js

```diff
@@ -169,8 +169,8 @@ const config = {
         copyright: `Copyright © ${new Date().getFullYear()} Biconomy Built with Docusaurus.`,
       },
       colorMode: {
-        defaultMode: 'dark',
-        disableSwitch: true,
+        defaultMode: 'light',
+        disableSwitch: false,
         respectPrefersColorScheme: false,
```

Comment:
i think for this case we should set respectPrefersColorScheme to be true that way it respects the users system preferences for light vs dark mode